pkg/cluster/reconcile.go

```diff
@@ -226,7 +226,6 @@ func (c *Cluster) disasterRecovery(left etcdutil.MemberSet) error {
 			return err
 		}
 	}
-	c.members = nil
```

Comment:
I looked at restoreSeedMember code, and understood that it calls startMember. I think we'd better to write a more extensive commit message to explain the code change when the context is not that clear.